index.html

```diff
@@ -1,3 +1,130 @@
+  -->
+  <!-- End Google Tag Manager (noscript) -->
+    <header>
+        <img class="avatar" src="/images/gazr.svg" alt="Gazr">
+        <p><strong>gazr</strong>, pronounced "Gather", is an opinionated way to launch <strong>common goals</strong> on your project.</p>  
+    </header>
+    <section>
+        <div class="box">
+            <h2 class="main-title">Main principles</h2>
+            <div class="box-content">
+                <ul>
+                    <li>
+                        A Container runtime as the <strong>only one</strong> dependency in developer laptop.
+                        <p class="details">
+                            Docker provides a full integration with <a href="https://www.docker.com/docker-mac">Mac</a>, <a href="https://www.docker.com/docker-windows">Windows</a> & <a href="https://www.docker.com/docker-ubuntu">Linux <em>(ubuntu)</em></a>.
+                        </p>
+                    </li>
+                    <li>
+                        Use generic tasks name to launch common actions.
+                    </li>
+                    <li>
+                        Let developers use the technologies & library they want.
+                    </li>
+                    <li>
+                        Simplify technical on-boarding.
+                    </li>
+                    <li>
+                        Simplify projects switching.
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="box box-grey">
+            <h2 class="main-title"><em>Makefile</em> specification</h2>
+            <ul class="make-specification">
+                <li>
+                    <span>make checkstyle</span>
+                    <p class="details">
+                        Check lint, checkstyle rules. e.g. <a href="https://www.pylint.org/">pylint</a>, <a href="https://github.com/squizlabs/PHP_CodeSniffer">phpcs</a>, <a href="https://eslint.org/">eslint</a>, <a href="https://github.com/checkstyle/checkstyle">checkstyle <em>(java)</em></a> etc ... 
+                    </p>
+                </li>
+                <li>
+                    <span>make complexity</span>
+                    <p class="details">
+                        Ciclomatic complexity check <em><a href="https://en.wikipedia.org/wiki/Cyclomatic_complexity">(McCabe)</a></em>, <a href="https://pypi.python.org/pypi/radon">radon <em>(python)</em></a>, <a href="https://eslint.org/docs/rules/complexity">eslint (js)</a>, <a href="https://phpmd.org/">PHPMD</a>, <a href="http://www.scalastyle.org/rules-0.5.0.html">rules</a> <em>(scala)</em> etc ...
+                    </p>
+                </li>
+                <li>
+                    <span>make fmt</span>
+                    <p class="details">
+                        Format code. e.g <a href="https://prettier.io/">Prettier <em>(js)</em></a>, <a href="https://golang.org/pkg/fmt/">fmt <em>(golang)</em></a>
```

Comment:
From PHPCodeSniffer you can use phpcbf to fix some (but not all) rules automatically (indent, alignments, etc.)